infrastructure/module/database/psql.helm_release.tf

```diff
@@ -46,12 +48,12 @@ resource "helm_release" "psql" {
 
   set {
     name  = "auth.username"
-    value = lookup(local.secret_data, "username", "user") //kubernetes_config_map.example.data.POSTGRESQL_USERNAME
```

Comment:
It's easier to remove local vars and do it like this:

lookup(jsondecode(sensitive(data.aws_secretsmanager_secret_version.current.secret_string)), "username", "what?")